core/relation/status.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package relation
+
+// Status describes the status of a relation.
+type Status string
+
+const (
+	Active  Status = "active"
+	Revoked Status = "revoked"
+)
```

Comment:
Thanks. A more detail on the `Status` docstring here as well as for `Active` and `Revoked` would go a long way.